packages/ui5-proxy-middleware/test/base/proxy.test.ts

```diff
@@ -0,0 +1,114 @@
+import { ui5Proxy } from '../../src';
+import * as hpm from 'http-proxy-middleware';
+import * as utils from '../../src/base/utils';
+import { ToolsLogger } from '@sap-ux/logger';
+
+describe('ui5Proxy', () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    test('ui5Proxy: default params', () => {
+        const createProxyMiddlewareSpy = jest.spyOn(hpm, 'createProxyMiddleware').mockImplementation(jest.fn());
+        const proxyResponseHandlerSpy = jest.spyOn(utils, 'proxyResponseHandler').mockImplementation(jest.fn());
+        const proxyRequestHandlerSpy = jest.spyOn(utils, 'proxyRequestHandler').mockImplementation(jest.fn());
+        const config = {
+            path: '/mypath',
+            url: 'https://example.com',
```

Comment:
Minor thing but our security expert recommend to use `.example` domains for all testing even if we never do a real request.

Reply:
Done